src/pages/components/cards.mdx

```diff
@@ -76,6 +76,47 @@ For information about required and optional elements, see the [content guidance
 
 ### Modifiers
 
+#### Logo
+
+The card can feature a logo at the top of the card. A logo can provide quick identification of a brand and be particularly useful for company success stories or case studies.
```

Comment:
```suggestion
The card can feature a logo at the top of the card. A logo can provide quick identification of a brand and be particularly useful for company success stories or case studies. For information about content in logo cards, see the
[content guidance table](#content-guidance-for-logo-card).
```
Just noticing the other card variants have an anchor link to the content guidance within their first paragraphs. 
Note: make sure the anchor works, I made a comment about changing the label for the content table to Logo card.